pkg/blockstorage/gcepd/gcepd.go

```diff
@@ -38,10 +40,11 @@ import (
 	"github.com/kanisterio/kanister/pkg/poll"
 )
 
-var _ blockstorage.Provider = (*gpdStorage)(nil)
-var _ zone.Mapper = (*gpdStorage)(nil)
+var _ blockstorage.Provider = (*GpdStorage)(nil)
+var _ zone.Mapper = (*GpdStorage)(nil)
 
-type gpdStorage struct {
+// GpdStorage implements blockstorage.Provider
+type GpdStorage struct {
 	service *compute.Service
 	project string
 }
```

Comment:
Why not expose the internal fields (or provide accessors) when you do this?

Reply:
Didn't need them. Only wanted access to the RegionToZoneMap. Can expose them as needed in the future.

Reply:
Ok